felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1638,19 +1667,60 @@ func (m *bpfEndpointManager) setJumpMapFD(ap attachPoint, fd bpf.MapFD) {
 	})
 }
 
-func (m *bpfEndpointManager) updatePolicyProgram(jumpMapFD bpf.MapFD, rules polprog.Rules) error {
+func (m *bpfEndpointManager) removePolicyDebugInfo(ifaceName string, hook tc.Hook) {
+	if !m.bpfPolicyDebugEnabled {
+		return
+	}
+	filename := bpf.PolicyDebugJSONFileName(ifaceName, string(hook))
+	os.Remove(filename)
```

Comment:
Think this can return an error, right to ignore that (what can we do?) but you should log that out